Basics/Calexp_guided_tour.ipynb

```diff
@@ -406,10 +640,28 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from lsst.geom.coordinates import Point2D\n",
+    "from lsst.geom import Point2D\n",
```

Comment:
It's probably better style to use `PointD` than `Point2D` (the 2 is implicit in the number of args)